cern-vdt/00-lsx.patch

```diff
@@ -0,0 +1,29 @@
+--- CMakeLists.txt	2022-03-16 00:11:50.000000000 +0800
++++ CMakeLists2.txt	2024-10-07 10:08:25.146943917 +0800
+@@ -15,12 +15,13 @@
+ option( AVX2 "Use AVX2 instruction set (default cache entry: OFF)" OFF)
+ option( FMA "Use FMA instruction set (default cache entry: OFF)" OFF)
+ option( USERFLAGS "Pass arbitrary flags to the compiler")
+-option( SSE "Use SSE instruction set (default cache entry: ON)" ON)
++option( SSE "Use SSE instruction set (default cache entry: ON)" OFF)
```

Comment:
It might be better to set this option in [cmake options](https://gitlab.archlinux.org/archlinux/packaging/packages/cern-vdt/-/blob/main/PKGBUILD?ref_type=heads#L32) and don't change upstream's default option.